test/testdata/generate.sh

```diff
@@ -136,6 +136,5 @@ scaffold_test_project project-v3 --plugins="go/v3"
 scaffold_test_project project-v4 --plugins="go/v4"
 scaffold_test_project project-v4-multigroup --plugins="go/v4"
 scaffold_test_project project-v4-declarative-v1 --plugins="go/v4,declarative"
-scaffold_test_project project-v4-config --component-config --plugins="go/v4"
```

Comment:
Should we change this to `scaffold_test_project project-v4-config --plugins="go/v4"` ? Just remove `component-config`

Reply:
this line scaffolds the sample under testdata with component config
So we are just removing here since we no longer need this scaffold